src/repr/src/scalar.rs

```diff
@@ -57,6 +57,8 @@ pub enum Datum<'a> {
     Bytes(&'a [u8]),
     /// A sequence of Unicode codepoints encoded as UTF-8.
     String(&'a str),
+    /// Like [`String`] but imposes a limit on the length of string supported.
+    Char(&'a str),
```

Comment:
Are you sure we need this? I'd expect `Datum::String` to do the trick for chars too, once you jam in some blanks?

Reply:
Removed `Datum::Char` and have everything working with `Datum::String`.